include/sys/ioctl.h

```diff
@@ -35,6 +35,10 @@ __BEGIN_DECLS
 #define TIOCGETA TCGETS
 #endif
 
+#ifndef IOCTL_ISO9660_GET_SECTOR
+#define IOCTL_ISO9660_GET_SECTOR 0x5601
```

Comment:
Maybe call it FIRST_SECTOR or START_SECTOR or LBA.
Just a SECTOR not very clear.

Reply:
Made it `IOCTL_ISO9660_GET_FIRST_EXTENT` to match with iso9660 terminology